js/src/features/project/components/ProjectRoleForm.tsx

```diff
@@ -57,176 +70,196 @@ export const ProjectRoleForm: React.FC<Props> = ({
-                      Add role
-                    </Button>
-                  </td>
-                </tr>
-              </tfoot>
-            </ProjectRoleFormTableWrapper>
+            <tr key={fieldIndex}>
+              {isOpenInput(fieldIndex) ? (
+                <td>
+                  <div className="role-form__input-wrapper">
+                    <Input
+                      defaultValue={field.name}
+                      ref={inputRef}
+                      onChange={(event) =>
+                        createRole(fieldIndex, event.target.value)
+                      }
+                      className="role-form__input"
+                    />
+                    {!!field.id && (
+                      <div className="role-form__controls-btn">
+                        <Button
+                          onClick={() => editInputRole(fieldIndex)}
+                          type="primary"
+                        >
+                          <CheckCircleOutlined />
+                        </Button>
+                        <Button
+                          onClick={() => setIndexOfEditableInput(-1)}
+                          type="danger"
+                        >
+                          <CloseCircleOutlined />
+                        </Button>
+                      </div>
+                    )}
+                  </div>
+                </td>
+              ) : (
+                <td>
+                  <div className="role-form__input-wrapper">
+                    <div className="role-form__description">{field.name}</div>
+                    <div className="role-form__controls-btn">
+                      <Button
+                        onClick={() => setIndexOfEditableInput(fieldIndex)}
+                        type="primary"
+                        className="role-form__controls-btn"
+                      >
+                        <EditOutlined />
+                      </Button>
+
+                      <Button
+                        onClick={() => removeRole(fieldIndex)}
+                        type="danger"
+                        className="role-form__controls-btn"
+                      >
+                        <DeleteOutlined />
+                      </Button>
+                    </div>
+                  </div>
+                </td>
+              )}
+              {rolePermissions.map((permission, index) => (
+                <td key={index}>
+                  <Checkbox
+                    onChange={() => toggleCheckbox(fieldIndex, permission)}
+                    checked={isChecked(permission, fieldIndex)}
+                    disabled={isDisabled(fieldIndex)}
+                  />
+                </td>
+              ))}
+            </tr>
           );
-        }}
-      </Form.List>
-    </Form>
+        })}
+      </tbody>
+      <tfoot>
+        <tr>
+          <td>
+            <Button onClick={() => addFormField()} type="primary">
```

Comment:
you can write `<Button onClick={addFormField} type="primary">`

Reply:
done